src/protocols/ssh/settings.c

```diff
@@ -217,6 +218,12 @@ enum SSH_ARGS_IDX {
      */
     IDX_BACKSPACE,
 
+    /**
+     * The terminal emulator type that is passed to programs (e.g. "xterm" or
```

Comment:
So, I have one minor nitpick about this comment - Guacamole is not actually passing the terminal emulator type to any particular program(s), this simply passes it to the system - it's up to the system to make it available to the programs.  Perhaps "passed to the remote system" would be better?